sopel/test_tools.py

```diff
@@ -122,18 +122,18 @@ def get_example_test(tested_func, msg, results, privmsg, admin,
                      owner, repeat, use_regexp, ignore=[]):
     """Get a function that calls ``tested_func`` with fake wrapper and trigger.
 
-    :param callable tested_func: a sopel callable that accepts a
-                                 ``SopelWrapper`` and a ``Trigger``.
-    :param str msg: message that is supposed to trigger the command.
-    :param list results: expected output from the callable.
-    :param bool privmsg: if true, make the message appear to have sent in a
+    :param callable tested_func: a Sopel callable that accepts a
+                                 ``SopelWrapper`` and a ``Trigger``
+    :param str msg: message that is supposed to trigger the command
+    :param list results: expected output from the callable
+    :param bool privmsg: if ``True``, make the message appear to have arrived in a
```

Comment:
Technically this line is now 82 characters. You can fix it during squash if you care. :D